minds/minds.py

```diff
@@ -261,6 +269,9 @@ def create(
         :param replace: if true - to remove existing mind, default is false
         :return: created mind
         """
+        
+        if not utils.validate_mind_name(name):
+            raise exc.MindNameInvalid("Mind name should start with a letter and contain only letters, numbers or underscore, with a maximum of 32 characters. Spaces are not allowed.")
```

Comment:
we can move this exception into validate_mind_name, to not duplicate messages

Reply:
Done